### PR TITLE
doc: update doc versions for 2.2.99-dev1 tag

### PR DESCRIPTION
**Pull request by umapraseeda**
Update doc versions for 2.2.99-dev1 tag

Signed-off-by: UMA PRASEEDA <uma.praseeda@nordicsemi.no>

- [x] Add highlights on the welcome page

### CODE REVIEW

File: doc/nrf/index.rst [hunk 1]
```diff
@@ -3,6 +3,14 @@
 Welcome to the |NCS|!
 #####################
 
+.. important::
+
+   |NCS| v2.2.99-dev1 is a development tag and will be replaced by v2.3.0 in the future.
+
+   The development tag contains the following major update:
+
+.. To be added
```
Comment (umapraseeda) [inline doc/nrf/index.rst:12]: @bama-nordic, @ioannisg, could you provide the input for this section?

File: doc/nrf/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -113,11 +111,11 @@ See `Zigbee samples`_ for the list of changes for the Zigbee samples.
 ESB
```
Comment (greg-fer) [inline doc/nrf/releases/release-notes-changelog.rst:111]:
```suggestion
Enhanced ShockBurst (ESB)
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 3]
```diff
@@ -113,11 +111,11 @@ See `Zigbee samples`_ for the list of changes for the Zigbee samples.
...
-  * Added support for front-end modules.
-  * The ESB module requires linking the :ref:`MPSL library <nrfxlib:mpsl_lib>`.
-  * The number of PPI/DPPI channels used is increased from 3 to 6.
-  * Assigned events 6-7 from the EGU0 instance to the ESB module.
-  * Changed the type parameter of the function :c:func:`esb_set_tx_power` to ``int8_t``.
+* Added support for front-end modules.
+* The ESB module requires linking the :ref:`MPSL library <nrfxlib:mpsl_lib>`.
+* The number of PPI/DPPI channels used is increased from 3 to 6.
+* Assigned events 6-7 from the EGU0 instance to the ESB module.
+* Changed the type parameter of the function :c:func:`esb_set_tx_power` to ``int8_t``.
```
Comment (greg-fer) [inline doc/nrf/releases/release-notes-changelog.rst:118]:
```suggestion
* Added support for front-end modules.
  The ESB module requires linking the :ref:`MPSL library <nrfxlib:mpsl_lib>`.

* Updated:

  * Number of PPI/DPPI channels used from 3 to 6.
  * Events 6-7 from the EGU0 instance by assigning them to the ESB module.
  * The type parameter of the function :c:func:`esb_set_tx_power` to ``int8_t``.
```
This line sounds like part of the support for FEM? Anyway, it has a formatting that is off for release notes.
```
* The ESB module requires linking the :ref:`MPSL library <nrfxlib:mpsl_lib>`.
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 4]
```diff
@@ -166,16 +163,18 @@ nRF5340 Audio
...
+
+  * An application log informing about the configuration option value update in the :ref:`nrf_desktop_motion`.
+  * Application-specific Kconfig options (:ref:`CONFIG_DESKTOP_LOG <config_desktop_app_options>` and :ref:`CONFIG_DESKTOP_SHELL<config_desktop_app_options>`) to simplify the debug configurations for the Logging and Shell subsystems.
+    See the debug configuration section of the :ref:`nrf_desktop` application for more details.
 
-* Changed:
+* Updated:
 
-   * Implemented adjustments to avoid flooding logs:
+  * Implemented adjustments to avoid flooding logs:
```
Comment (greg-fer) [inline doc/nrf/releases/release-notes-changelog.rst:174]:
```suggestion
  * Implemented the following adjustments to avoid flooding logs:
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 5]
```diff
@@ -256,19 +248,13 @@ nRF9160 samples
...
 
-  * Added:
-
-    * Usage of GCI search option if running modem firmware 1.3.4.
-
-  * Updated:
-
-    * The sample now waits for RRC idle mode before requesting neighbor cell measurements.
+  * Added the usage of GCI search option if running modem firmware version 1.3.4.
+  * Updated the sample so that the sample now waits for RRC idle mode before requesting neighbor cell measurements.
```
Comment (greg-fer) [inline doc/nrf/releases/release-notes-changelog.rst:257]:
```suggestion
  * Updated the sample, so that it now waits for RRC idle mode before requesting neighbor cell measurements.
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 6]
```diff
@@ -256,19 +248,13 @@ nRF9160 samples
...
 * :ref:`nrf_cloud_rest_cell_pos_sample` sample:
 
-  * Added:
-
-    * Usage of GCI search option if running modem firmware 1.3.4.
-
-  * Updated:
-
-    * The sample now waits for RRC idle mode before requesting neighbor cell measurements.
+  * Added the usage of GCI search option if running modem firmware version 1.3.4.
```
Comment (greg-fer) [inline doc/nrf/releases/release-notes-changelog.rst:256]:
```suggestion
  * Added the usage of GCI search option if running modem firmware v1.3.4.
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 7]
```diff
@@ -514,8 +498,7 @@ Other libraries
 
 * PCM Mix:
```
Comment (greg-fer) [inline doc/nrf/releases/release-notes-changelog.rst:499]:
```suggestion
* :ref:`lib_pcm_mix` library:
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 8]
```diff
@@ -514,8 +498,7 @@ Other libraries
...
 
-  * PCM mix (Pulse Code Modulation) audio mixer has been moved out of the nRF5340 Audio
-    application, and into lib/pcm_mix.
+  * PCM mix (Pulse Code Modulation) audio mixer has been moved out of the :ref:`nrf53_audio_app` application and into :file:`lib/pcm_mix`.
```
Comment (greg-fer) [inline doc/nrf/releases/release-notes-changelog.rst:501]:
```suggestion
  * New library.
    This was previously a component of the :ref:`nrf53_audio_app` application, now moved to :file:`lib/pcm_mix`.
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 9]
```diff
@@ -168,16 +167,18 @@ nRF5340 Audio
 nRF Desktop
 -----------
 
-* Added an application log informing about the configuration option value update in the :ref:`nrf_desktop_motion`.
-* Added application-specific Kconfig options (:ref:`CONFIG_DESKTOP_LOG <config_desktop_app_options>` and :ref:`CONFIG_DESKTOP_SHELL<config_desktop_app_options>`) to simplify the debug configurations for the Logging and Shell subsystems.
-  See the debug configuration section of the :ref:`nrf_desktop` application for more details.
+* Added:
 
-* Changed:
+  * An application log informing about the configuration option value update in the :ref:`nrf_desktop_motion`.
```
Comment (b-gent) [inline doc/nrf/releases/release-notes-changelog.rst:172]: I would rephrase this to prevent this massive compound noun, e.g.
"An application log informing that the value of a configuration option has been updated"

**Review by b-gent — CHANGES_REQUESTED** [review]

File: doc/nrf/releases/release-notes-changelog.rst [hunk 10]
```diff
@@ -75,16 +75,14 @@ Matter
...
-  * Connection timing out when attaching to a Wi-Fi Access Point that requires Wi-Fi Protected Access 3 (WPA3).
-
 * Updated:
 
-  * Default heap implementation to use Zephyr's ``sys_heap`` (:kconfig:option:`CONFIG_CHIP_MALLOC_SYS_HEAP`) to better control the RAM usage of Matter applications.
+  * The default heap implementation to use Zephyr's ``sys_heap`` (:kconfig:option:`CONFIG_CHIP_MALLOC_SYS_HEAP`) to better control the RAM usage of Matter applications.
   * :ref:`ug_matter_device_certification` page with a section about certification document templates.
   * :ref:`ug_matter_overview_commissioning` page with information about :ref:`ug_matter_network_topologies_commissioning_onboarding_formats`.
 
+* Fixed the issue of connection timing out when attaching to a Wi-Fi Access Point that requires Wi-Fi Protected Access 3 (WPA3).
```
Comment (peknis) [inline doc/nrf/releases/release-notes-changelog.rst:84]: Do we need the capitalization of Access Point? I'd say no.

File: doc/nrf/releases/release-notes-changelog.rst [hunk 11]
```diff
@@ -110,14 +108,16 @@ Zigbee
...
+  The ESB module requires linking the :ref:`MPSL library <nrfxlib:mpsl_lib>`.
+* Updated:
 
-  * Added support for front-end modules.
-  * The ESB module requires linking the :ref:`MPSL library <nrfxlib:mpsl_lib>`.
-  * The number of PPI/DPPI channels used is increased from 3 to 6.
-  * Assigned events 6-7 from the EGU0 instance to the ESB module.
-  * Changed the type parameter of the function :c:func:`esb_set_tx_power` to ``int8_t``.
+  * Number of PPI/DPPI channels used from 3 to 6.
+  * Events 6-7 from the EGU0 instance by assigning them to the ESB module.
```
Comment (peknis) [inline doc/nrf/releases/release-notes-changelog.rst:119]:
```suggestion
  * Events 6 and 7 from the EGU0 instance by assigning them to the ESB module.
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 12]
```diff
@@ -110,14 +108,16 @@ Zigbee
...
+* Added support for front-end modules.
+  The ESB module requires linking the :ref:`MPSL library <nrfxlib:mpsl_lib>`.
+* Updated:
 
-  * Added support for front-end modules.
-  * The ESB module requires linking the :ref:`MPSL library <nrfxlib:mpsl_lib>`.
-  * The number of PPI/DPPI channels used is increased from 3 to 6.
-  * Assigned events 6-7 from the EGU0 instance to the ESB module.
-  * Changed the type parameter of the function :c:func:`esb_set_tx_power` to ``int8_t``.
+  * Number of PPI/DPPI channels used from 3 to 6.
```
Comment (peknis) [inline doc/nrf/releases/release-notes-changelog.rst:118]: Should we have the numbers in writing here? -> from three to six

File: doc/nrf/releases/release-notes-changelog.rst [hunk 13]
```diff
@@ -168,16 +167,18 @@ nRF5340 Audio
...
 -----------
 
-* Added an application log informing about the configuration option value update in the :ref:`nrf_desktop_motion`.
-* Added application-specific Kconfig options (:ref:`CONFIG_DESKTOP_LOG <config_desktop_app_options>` and :ref:`CONFIG_DESKTOP_SHELL<config_desktop_app_options>`) to simplify the debug configurations for the Logging and Shell subsystems.
-  See the debug configuration section of the :ref:`nrf_desktop` application for more details.
+* Added:
 
-* Changed:
+  * An application log informing about the configuration option value update in the :ref:`nrf_desktop_motion`.
+  * Application-specific Kconfig options (:ref:`CONFIG_DESKTOP_LOG <config_desktop_app_options>` and :ref:`CONFIG_DESKTOP_SHELL<config_desktop_app_options>`) to simplify the debug configurations for the Logging and Shell subsystems.
```
Comment (peknis) [inline doc/nrf/releases/release-notes-changelog.rst:173]:
```suggestion
  * Application-specific Kconfig options (:ref:`CONFIG_DESKTOP_LOG <config_desktop_app_options>` and :ref:`CONFIG_DESKTOP_SHELL <config_desktop_app_options>`) to simplify the debug configurations for the Logging and Shell subsystems.
```
Do we need to have the Kconfig options in the brackets? I would leave the brackets out.

**Review by peknis — CHANGES_REQUESTED** [review]

File: doc/nrf/releases/release-notes-changelog.rst [hunk 14]
```diff
@@ -168,16 +167,18 @@ nRF5340 Audio
...
 -----------
 
-* Added an application log informing about the configuration option value update in the :ref:`nrf_desktop_motion`.
-* Added application-specific Kconfig options (:ref:`CONFIG_DESKTOP_LOG <config_desktop_app_options>` and :ref:`CONFIG_DESKTOP_SHELL<config_desktop_app_options>`) to simplify the debug configurations for the Logging and Shell subsystems.
-  See the debug configuration section of the :ref:`nrf_desktop` application for more details.
+* Added:
 
-* Changed:
+  * An application log informing that the value of a configuration option has been updated in the :ref:`nrf_desktop_motion`.
+  * Application-specific Kconfig options :ref:`CONFIG_DESKTOP_LOG <config_desktop_app_options>` and :ref:`CONFIG_DESKTOP_SHELL<config_desktop_app_options>` to simplify the debug configurations for the Logging and Shell subsystems.
```
Comment (peknis) [inline doc/nrf/releases/release-notes-changelog.rst:173]:
```suggestion
  * Application-specific Kconfig options :ref:`CONFIG_DESKTOP_LOG <config_desktop_app_options>` and :ref:`CONFIG_DESKTOP_SHELL <config_desktop_app_options>` to simplify the debug configurations for the Logging and Shell subsystems.
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Comment by umapraseeda** [comment]
@raya006 @mjd90 the links to the PS and the application notes would appear here: https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-9931/nrf/ug_wifi.html .

**Review by gmarull — APPROVED** [review]

File: doc/nrf/releases/release-notes-changelog.rst [hunk 15]
```diff
@@ -121,14 +124,16 @@ Zigbee
...
+-------------------------
 
-  * Added support for front-end modules.
-  * The ESB module requires linking the :ref:`MPSL library <nrfxlib:mpsl_lib>`.
-  * The number of PPI/DPPI channels used is increased from 3 to 6.
-  * Assigned events 6-7 from the EGU0 instance to the ESB module.
-  * Changed the type parameter of the function :c:func:`esb_set_tx_power` to ``int8_t``.
+* Added support for front-end modules.
+  The ESB module requires linking the :ref:`MPSL library <nrfxlib:mpsl_lib>`.
+* Updated:
```
Comment (peknis) [inline doc/nrf/releases/release-notes-changelog.rst:132]:
```suggestion
  The ESB module requires linking the :ref:`MPSL library <nrfxlib:mpsl_lib>`.
  
* Updated:
```

File: doc/nrf/releases/release-notes-changelog.rst [hunk 16]
```diff
@@ -191,18 +194,21 @@ nRF Machine Learning (Edge Impulse)
...
 -----------
 
-* Removed separate configurations enabling :ref:`zephyr:shell_api` (:file:`prj_shell.conf`).
-  Shell support can be enabled for a given configuration with the single Kconfig option (:ref:`CONFIG_DESKTOP_SHELL <config_desktop_app_options>`).
-* Added an application log informing about the configuration option value update in the :ref:`nrf_desktop_motion`.
-* Added application-specific Kconfig options (:ref:`CONFIG_DESKTOP_LOG <config_desktop_app_options>` and :ref:`CONFIG_DESKTOP_SHELL<config_desktop_app_options>`) to simplify the debug configurations for the Logging and Shell subsystems.
-  See the debug configuration section of the :ref:`nrf_desktop` application for more details.
+* Added:
+
+  * An application log informing that the value of a configuration option has been updated in the :ref:`nrf_desktop_motion`.
```
Comment (peknis) [inline doc/nrf/releases/release-notes-changelog.rst:199]:
```suggestion
  * An application log indicating that the value of a configuration option has been updated in the :ref:`nrf_desktop_motion`.
```

**Review by peknis — CHANGES_REQUESTED** [review]

**Review by peknis — APPROVED** [review]

**Review by bama-nordic — APPROVED** [review]
LGTM

**Review by b-gent — APPROVED** [review]

File: doc/versions.json [hunk 17]
```diff
@@ -43,6 +44,15 @@
       "tfm": "1.6.0",
       "matter": ""
     },
+    "2.2.99-dev1": {
```
Comment (umapraseeda) [inline doc/versions.json:47]: fyi: https://github.com/nrfconnect/sdk-nrf/pull/10683/files
